### ISSUE TITLE
A11y: Header search form does not have correct label

### ISSUE THREAD

**Issue by MangopearUK** [MangopearUK/European-Boating-Association--Theme#385]
Failing Elements
--

`<input class="o-form__input" type="search" name="s" id="header-search" value="" placeholder="Search this website">`
